cmd/agent/dist/conf.d/disk.d/conf.yaml.default

```diff
@@ -4,6 +4,33 @@
 
 init_config:
 
+    ## @param file_system_global_exclude - list of strings - optional 
+    ## Instruct the check to always add these patterns to `file_system_exclude`.
+    ##
+    ## WARNING: Overriding these defaults could negatively impact your system or
+    ## the performance of the check.
+    #
+    # file_system_global_exclude:
+    #   - iso9660$
+    #   - tracefs$
```

Comment:
I don't see in the code where were we set this default values. Could you point me where are we setting the default values? Same for `mount_point_global_exclude `